src/App/BrownBag/View.elm

```diff
@@ -4,18 +4,18 @@ import Html exposing (..)
 import Html.Attributes exposing (..)
 import Common.Utils exposing (brandUrl)
 import App.BrownBag.Messages exposing (Msg(..))
-import App.BrownBag.Models exposing (Presenter, Status(..))
+import App.BrownBag.Models exposing (BrownBag, Status(..))
```

Comment:
@akabiru We have a bunch of functions and variables here with `presenter` somewhere in their names. Would you advise renaming them? Or do they make sense as they are?